package.json

```diff
@@ -16,7 +16,7 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.3.5",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
Nice job updating the version! 🎉 However, if there were any breaking changes between versions, make sure to test the project fully and update any necessary code.